Dockerfile.ubuntu

```diff
@@ -123,16 +123,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
             vim \
             xz-utils && \
             apt-get clean && \
-            rm -rf /var/lib/apt/lists/*
-
-# Python <3.10 needed to build TFS in current version
-RUN if [ "$BASE_IMAGE" == "ubuntu:22.04" ] ; then \ 
-        apt-get update && apt-get install -y software-properties-common gpg gpg-agent --no-install-recommends && \
-        add-apt-repository ppa:deadsnakes/ppa && \
-        apt-get install -y python3.8 python3.8-dev python3.8-distutils --no-install-recommends && \
-        apt-get clean && rm -rf /var/lib/apt/lists/* && \
-        update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 1 ; \
-    fi ; \
+            rm -rf /var/lib/apt/lists/* && \
     python3 -m pip install numpy==1.21.0 --no-cache-dir
```

Comment:
Isn't numpy needed for tensorflow build? Shouldn't we have it with Python 3.8?

Reply:
Its needed for ovms_dependencies.